docs/whats-new.md

```diff
@@ -1,8 +1,21 @@
 # What's New
 
+## v6.2
+
+Release date: TBD April, 2019
```

Comment:
@ibgreen Looks like these dates / versions are wrong? https://github.com/uber-common/viewport-mercator-project/releases

Reply:
Noted. I took care of that in the v6.2.0 publish commit (pushed to master).

@jckr is working on a gatsby website update, when published that will pick up the correct date.